.gitignore

```diff
@@ -19,6 +19,7 @@ lib64/
 parts/
 sdist/
 var/
+.vscode/
```

Comment:
this is fine, but to make it easier to keep track of what we added (above the GitHub default), can you add this to one of the sections below https://github.com/rlworkgroup/garage/pull/764/files#diff-a084b794bc0759e7a6b77810e01874f2R120

you might consider sending a PR to https://github.com/github/gitignore